### ISSUE TITLE
Error after click "Save Changes"

### ISSUE THREAD

**Issue by vitaliykononov** [evertiro/historical-redux2#6]
I am tried few times setup framework from fresh download, but get error when trying to save options.

Steps to reproduse issue:
1) download framework, and connect with theme 
2) Open "Options" section from 
3) Click "Save options"

Notice: Uninitialized string offset: 0 in .../admin/options/validation/color/validation_color.php on line 44

Warning: Cannot modify header information - headers already sent by (output started at .../wp-content/themes/fatwp/admin/options/validation/color/validation_color.php:44) in ..../wp-includes/pluggable.php on line 881

**Comment by evertiro** [comment]
My fault, I was working on the color picker and must have screwed it up. I'll push a fix shortly.

**Comment by evertiro** [comment]
Hmm... I haven't been able to reproduce this actually, all my tests are succeeding. Are you using the demo or are you modifying it?

**Comment by vitaliykononov** [comment]
sorry working now. I think problem was after I changed NHP to Redux. I make a clear instalation of Wordpress and now its working.

One more issue I've found in framework description on main page https://github.com/ghost1227/Redux-Framework

Simply include the options.php file in your theme or plugin, like so:

get_template_part('redux', 'options');

But when I download framework there is file redux-framework.php

so we must have something like this

get_template_part('redux', 'framework');

Thanks
